setup.py

```diff
@@ -269,24 +269,23 @@ def check_linker_need_libatomic():
 DEFINE_MACROS = (('_WIN32_WINNT', 0x600),)
 asm_files = []
 NO_BORING_ASM = True
```

Comment:
If we use `asm_key`, then there is no use for `NO_BORING_ASM`.

Reply:
correct. removed.